packages/gatsby/src/utils/feedback.ts

```diff
@@ -22,6 +25,19 @@ export function showFeedbackRequest(): void {
   report.log(`\nGive us your feedback here: https://gatsby.dev/feedback\n\n`)
 }
 
+export function showSevenDayFeedbackRequest(): void {
+  getConfigStore().set(sevenDayKey, Date.now())
+  trackCli(`SHOW_SEVEN_DAY_FEEDBACK_LINK`, {
+    name: `https://www.surveymonkey.com/r/gatsbyjs-feedback-cli`,
```

Comment:
Let's use a link similar to the other survey: `https://gatsby.dev/feedback-survey`

Reply:
Already in there!